ci/build.ps1

```diff
@@ -98,27 +99,17 @@ npm.cmd install -g neovim
 Get-Command -CommandType Application neovim-node-host.cmd
 npm.cmd link neovim
 
-#npm.cmd install -g tree-sitter-cli
-#npm.cmd link tree-sitter-cli
+npm.cmd install -g 'tree-sitter-cli@0.15.9'
 
-mkdir c:\treesitter
-$env:TREE_SITTER_DIR = "c:\treesitter"
-#$env:PATH = "c:\treesitter;$env:PATH"
+$env:TREE_SITTER_DIR = $env:USERPROFILE + "\tree-sitter-build"
+mkdir "$env:TREE_SITTER_DIR\bin"
 $client = new-object System.Net.WebClient
-cd c:\treesitter
 
-if ($bits -eq 32) {
-  $client.DownloadFile("https://github.com/tree-sitter/tree-sitter/releases/download/0.15.9/tree-sitter-windows-x86.gz", "c:\treesitter\tree-sitter-cli.gz")
-}
-elseif ($bits -eq 64) {
-  $client.DownloadFile("https://github.com/tree-sitter/tree-sitter/releases/download/0.15.9/tree-sitter-windows-x64.gz", "c:\treesitter\tree-sitter-cli.gz")
-}
-python -c "import gzip, shutil; f1,f2 = gzip.open('tree-sitter-cli.gz', 'rb'), open('tree-sitter.exe', 'wb'); shutil.copyfileobj(f1, f2); f2.close()"
-
-$client.DownloadFile("https://codeload.github.com/tree-sitter/tree-sitter-c/zip/v0.15.2","c:\treesitter\tree_sitter_c.zip")
-Expand-Archive c:\treesitter\tree_sitter_c.zip -DestinationPath c:\treesitter\
-cd c:\treesitter\tree-sitter-c-0.15.2
-c:\treesitter\tree-sitter.exe test
+$client.DownloadFile("https://codeload.github.com/tree-sitter/tree-sitter-c/zip/v0.15.2",".\tree_sitter_c.zip")
+Expand-Archive .\tree_sitter_c.zip -DestinationPath .
+cd tree-sitter-c-0.15.2
+tree-sitter.cmd test
```

Comment:
I guess this .cmd is coming from the node package, can't find it in the zip from https://codeload.github.com/tree-sitter/tree-sitter-c/zip/v0.15.2 . Is there a reason not to use the .exe? Does invoking `.cmd` directly in powershell potentially have problems?

Reply:
nodejs creates shell shims for "executable" scripts so the user can run it in the shell. nodejs doesn't expose 3rd-party `.exe` in `PATH` by default aside from the shipped binaries (ie `node.exe`).